### ISSUE TITLE
Multiple sources in the targets file

### ISSUE THREAD

**Issue by dennismi** [davidfowl/NuGetPowerTools#15]
is this possible?

**Comment by davidfowl** [comment]
Yep. Just tweak the `<PackageSoures>` element in the targets file:

```
    <PackageSources>c:\foo;http://mycustomfeed</PackageSources>
```

Those arguments get passed to nuget.exe with the -source parameter

**Comment by dennismi** [comment]
great, couldn't find the syntax for multiple sources :) 

thx

**Comment by dennismi** [comment]
Ok, need to reopen this one .. :) 

My PackageSources is: 

```
<PackageSources>https://go.microsoft.com/fwlink/?LinkID=206669;\\dkstole-tfs2010\packages</PackageSources>
```

and the output of the build process says this: 
error MSB3073: The command ""D:\@development\Tolerance\main\source.nuget\nuget.exe" install "D:\@development\Tolerance\main\source\ServiceSikring\packages.config" -source "http://nuget/nuget" -o "D:\@development\Tolerance\main\source\packages"" exited with code 1.

where the source option isn't picking up the two sources from the targets file, any idea why?

regards
dennis

**Comment by dennismi** [comment]
and removing the node itself from the .targets file has no effect :(

is there some caching involved?

**Comment by dennismi** [comment]
it does cache the targets file, and need to close and open VS to make it work ...
